expression/builtin_miscellaneous.go

```diff
@@ -289,7 +289,52 @@ type builtinIsIPv4Sig struct {
 
 // See https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv4
 func (b *builtinIsIPv4Sig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("IS_IPV4")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return types.Datum{}, errors.Trace(err)
+	}
+	if args[0].IsNull() {
+		d.SetInt64(0)
+		return d, nil
+	}
+	// isIPv4(str)
+	// args[0] string
+	s, err := args[0].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	if mysqlIsIPv4(s) {
+		d.SetInt64(1)
+	} else {
+		d.SetInt64(0)
+	}
+	return d, nil
+}
+
+// mysql is_ipv4 don't accesp IPv6 expressed IPv4 format
+func mysqlIsIPv4(ip string) bool {
```

Comment:
mysql 'is_ipv4' works differently in some way with package 'net' or UNIX system-calls. So I add "mysql" before "isIPv4" to emphasis that this function represent mysql's behavior